plots/dropsonde_overview_map.py

```diff
@@ -0,0 +1,175 @@
+# %%
+import xarray as xr
+import numpy as np
+import matplotlib.pyplot as plt
+import cartopy.feature
+import cartopy.crs as ccrs
+import settings
+
+from orcestra import get_flight_segments
+
+# %%
+l3 = f"{settings.root}/products/HALO/dropsondes/Level_3/PERCUSION_Level_3.zarr"
+dsdrop = xr.open_dataset(l3, engine="zarr")  # .swap_dims({"sonde_id": "launch_time"})
```

Comment:
```suggestion
dsdrop = xr.open_dataset(l3, engine="zarr")
```

I would delete the comment